wacruit/src/database/migrations/versions/2023_08_16_0132-d8d10f453a15_recruiting_application.py

```diff
@@ -0,0 +1,61 @@
+"""Create table recruiting_application
+
+Revision ID: d8d10f453a15
+Revises: 5bbc8b8ae19c
+Create Date: 2023-08-16 01:32:17.979189
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = "d8d10f453a15"
+down_revision = "5bbc8b8ae19c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "recruiting_application",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("user_id", sa.Integer(), nullable=False),
+        sa.Column("recruiting_id", sa.Integer(), nullable=False),
+        sa.Column(
+            "status",
+            sa.Enum(
+                "IN_PROGRESS",
+                "ACCEPTED",
+                "REJECTED",
+                name="recruitingapplicationstatus",
+            ),
+            server_default=sa.text("'IN_PROGRESS'"),
+            nullable=False,
+        ),
+        sa.Column(
+            "created_at",
+            sa.DateTime(timezone=True),
+            server_default=sa.text("CURRENT_TIMESTAMP"),
+            nullable=False,
+        ),
+        sa.Column(
+            "updated_at",
+            sa.DateTime(timezone=True),
+            server_default=sa.text("CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP"),
+            nullable=False,
+        ),
+        sa.ForeignKeyConstraint(
+            ["recruiting_id"], ["recruiting.id"], ondelete="CASCADE"
+        ),
+        sa.ForeignKeyConstraint(["user_id"], ["user.id"], ondelete="CASCADE"),
+        sa.PrimaryKeyConstraint("id", "user_id", "recruiting_id"),
+        sa.UniqueConstraint("user_id", "recruiting_id", name="ak_user_recruiting"),
```

Comment:
@shp7724 각 필드에 정의하면 이렇게 2개의 unique constraint가 만들어져요

```suggestion
        sa.UniqueConstraint("recruiting_id"),
        sa.UniqueConstraint("user_id"),
```

Reply:
지금처럼 composite 으로 했을 경우 `show create table` 결과
``` mysql
  UNIQUE KEY `ak_user_recruiting` (`user_id`,`recruiting_id`),
```

각 필드에 정의했을 경우 `show create table` 결과
``` mysql
  UNIQUE KEY `recruiting_id` (`recruiting_id`),
  UNIQUE KEY `user_id` (`user_id`),
```

Reply:
아하.. 내 기억이 잘못돼있었나봄. 
굳굳